docs/userguide/configuring.rst

```diff
@@ -305,8 +305,8 @@ Amazon Web Services
 .. image:: ./aws_image.png
 
 .. note::
-   Please note that **boto3** library must be installed to use AWS with Parsl.
-   This can be installed via ``python3 -m pip install parsl[aws]``
+   Dependencies for using AWS with Parsl can be
+   installed via ``python3 -m pip install parsl[aws]``
```

Comment:
I wonder if it would be better to turn this around: "To use AWS with Parsl, install Parsl with AWS dependencies via ``python3 -m pip install parsl[aws]``"

Reply:
done